flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -421,16 +427,29 @@ public void dispose() {
 			// invalidate the reference
 			db = null;
 
-			IOUtils.closeQuietly(columnOptions);
 			IOUtils.closeQuietly(dbOptions);
 			IOUtils.closeQuietly(writeOptions);
+
+			for (RocksDbKvStateInfo kvStateInfo : kvStateInformation.values()) {
+				IOUtils.closeQuietly(kvStateInfo.compactionFilter);
```

Comment:
what is the order of these `close` should follow ?  `columnFamily` -> `columnOptions` -> `db` -> `dbOptions` -> `writeOptions` -> `filter` ?    Can this be put in a single function avoiding missing ~?

Reply:
The closing order should be reverse to the order in which objects were created. What do you mean by `missing ~`?